app/presenters/completions_presenter.rb

```diff
@@ -120,6 +123,10 @@ def pii
     end
   end
 
+  def multiple_emails?
+    current_user.confirmed_email_addresses.count > 1
```

Comment:
So for this one- I had changed it to that but it caused the Add email link to not show. It depends on the count being higher than 1 confirmed email address or it defaults to Change.

Reply:
Ah, that makes sense. `any?` would basically be equivalent to `count > 0`, not `count > 1`. I know that `many?` exists as well, dunno if it's any more performance than comparing the count.

```suggestion
    current_user.confirmed_email_addresses.many?
```

Reply:
It looks like it could be, since [it internally includes `LIMIT 2` as part of the SQL query](https://guides.rubyonrails.org/active_record_querying.html#:~:text=You%20can%20also%20use%20any%3F%20and%20many%3F%20to%20check%20for%20existence%20on%20a%20model%20or%20relation.%20many%3F%20will%20use%20SQL%20count%20to%20determine%20if%20the%20item%20exists.) to avoid querying everything.